src/state-card-custom-ui.html

```diff
@@ -30,36 +30,76 @@
     'inputChanged(hass, inDialog, stateObj)',
   ],
 
-  badgeMode: function (hass, stateObj) {
+  badgeMode: function (hass, stateObj, badgeLabelBorder, badgeLabelText, badgeBackground) {
     var states = [];
-    stateObj.attributes.entity_id.forEach((id) => {
-      var state = hass.states[id];
-      if (!state) {
-        /* eslint-disable no-console */
-        console.warn('Unknown ID ' + id + ' in group ' + stateObj.entity_id);
-        /* eslint-enable no-console */
-        return;
-      }
-      if (!stateObj.attributes.badges_list ||
-          stateObj.attributes.badges_list.indexOf(state.entity_id) >= 0) {
-        states.push(window.customUI.maybeChangeObject(
-          this, state, false /* inDialog */, false /* allowHidden */));
-      }
-    });
+    if ( typeof stateObj.attributes.entity_id !== 'undefined' && window.hassUtil.computeDomain(stateObj) === 'group' ) {
```

Comment:
In the call of `badgeMode` you are passing in the domain, but here you compute it again.

Reply:
You're right. Makes no sense. Left-over from a little experiment. Should be removed.